README.rst

```diff
@@ -1,9 +1,31 @@
+.. image:: https://travis-ci.com/zopefoundation/zc.recipe.testrunner.svg?branch=master
+        :target: https://travis-ci.org/zopefoundation/zc.recipe.testrunner
+
+.. image:: https://coveralls.io/repos/github/zopefoundation/zc.recipe.testrunner/badge.svg?branch=master
+        :target: https://coveralls.io/github/zopefoundation/zc.recipe.testrunner?branch=master
+
+.. image:: https://img.shields.io/pypi/v/zc.recipe.testrunner.svg
+        :target: https://pypi.org/project/zc.recipe.testrunner/
+        :alt: Current version on PyPI
+
+.. image:: https://img.shields.io/pypi/pyversions/zc.recipe.testrunner.svg
+        :target: https://pypi.org/project/zc.recipe.testrunner/
+        :alt: Supported Python versions
+
 ******************
 Test-Runner Recipe
 ******************
```

Comment:
By inserting some images above this you changed it from being the ReStructuredText document title to being a simple ReST header.  Was that intentional?

Reply:
TIL. Going to fix it.